artemis-commons/src/main/java/org/apache/activemq/artemis/utils/actors/ProcessorBase.java

```diff
@@ -111,7 +141,10 @@ public final boolean isFlushed() {
 
    protected void task(T command) {
       tasks.add(command);
-      startPoller();
+      //there is no need to alert the poller if already in the event loop
```

Comment:
@clebertsuconic This is the improvement I've mentioned: I've left it here just to understand if it is safe but I can push in another PR apart (in the future) too, if it seems safe.